DTS-STN/Alpha-Site
# Home page banner

# Description

[Landing page hero image and title] https://trello.com/c/QWkOCj4B/57-landing-page-hero-image-and-title

Added a banner to look as close as possible to the https://www.canada.ca/en.html banner.

For the hard-coded strings as English, we will need to change it when i18n is incorporated into the project.

## Acceptance Criteria
Content AC:
 - consists of a title, followed by short byline
 - the site title and byline appear in a dark blue box
 - the site title is presented as an H1, with a red horizontal rule underneath
 - the byline appears immediately below the site title

Image AC:
 - images must be photographic (no illustration or abstract imagery)
 - choose images that show seasonal details (e.g. snow in winter, flowers in spring)
 - avoid images of people or faces as these are distracting
 - the banner image appears across the full width of the page, underneath the site header and behind the box containing the site title and headline

## Test Instructions

1. Navigate to branch called: homePageBanner
2. runthe comand: npm run dev
3.  see the banner on the home page.
4. Test different width including mobile.

## Help Requested

Any feedback is greatly appreciated. I am new to JavaScript unit test and Storybook.

## Checklist

- [ x] Unit tests have been added/updated

## Product and Sprint Backlog

https://trello.com/c/QWkOCj4B/57-landing-page-hero-image-and-title
